tests/test_comm.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+import numpy as np
+
+from tike.communicators import Comm
+
+
+class TestComm(unittest.TestCase):
+
+    def setUp(self, workers=4):
+        self.comm = Comm(workers)
+        self.xp = self.comm.pool.xp
+
+    def test_reduce(self):
+        if self.comm.pool.device_count < 2:
+            return  # skip test if only one device
```

Comment:
```suggestion
```
This tests should still work with less than 2 devices, right? If you agree, just click the "commit suggestion" button to accept the suggestion. You don't need to make a commit manually.